akka-actor/src/main/scala/akka/actor/CoordinatedShutdown.scala

```diff
@@ -421,6 +427,50 @@ final class CoordinatedShutdown private[akka] (
   def addTask(phase: String, taskName: String, task: Supplier[CompletionStage[Done]]): Unit =
     addTask(phase, taskName)(() => task.get().toScala)
 
+  /**
+   * Scala API: Add an actor termination task to a phase. It doesn't remove
+   * previously added tasks. Tasks added to the same phase are executed in
+   * parallel without any ordering assumptions. Next phase will not start until
+   * all tasks of previous phase have been completed.
+   *
+   * When executed, this task will first send the given stop message, if defined,
+   * to the actor, then it will watch the actor, and complete when the actor
+   * terminates.
+   *
+   * Tasks should typically be registered as early as possible after system
+   * startup. When running the coordinated shutdown tasks that have been registered
+   * will be performed but tasks that are added too late will not be run.
+   * It is possible to add a task to a later phase by a task in an earlier phase
+   * and it will be performed.
+   */
+  def addActorTerminationTask(phase: String, taskName: String, actor: ActorRef, stopMsg: Option[Any] = None): Unit =
```

Comment:
maybe remove the default value of `stopMsg`? could be good to make it clear that it is a user decision